src/modules/Rotate/Module.js

```diff
@@ -23,20 +23,17 @@ module.exports = function Rotate(options, UI) {
 
     function extraManipulation(pixels) {
       var rotate_value = (options.rotate) % 360;
-
-      if (rotate_value % 360 == 0)
-        return pixels;
-
-      var bitmap = new imagejs.Bitmap({ width: pixels.shape[0], height: pixels.shape[1] });
-      bitmap._data.data = pixels.data;
-
-      var rotated = bitmap.rotate({
-        degrees: rotate_value,
-      });
-      pixels.data = rotated._data.data;
-
+      var radians = 3.141592653589793 * rotate_value / 180;
+      var width = pixels.shape[0];
+      var height = pixels.shape[1];
+      var cos = Math.cos(radians);
```

Comment:
If you have split this into two files, is this still required?

Reply:
@HarshKhandeparkar i think these would still be required as we are returning pixels2 from the Rotate.js ,so it must be initialised before passing it as argument in module.js.
What do you think about this?